src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1574,6 +1574,13 @@ private function export($value)
             return $dirname;
         }
 
+        if (is_string($value) && false !== strpos($value, "\n")) {
+            $cleanParts = explode("\n", $value);
+            $cleanParts = array_map(function ($part) { return var_export($part, true); }, $cleanParts);
+
+            return implode(' . "\n" . ', $cleanParts);
```

Comment:
spaces around dots should be remove I think, per our CS policy.

Reply:
@nicolas-grekas sure, already done.